frontend/src/common/models/user/user.ts

```diff
@@ -3,14 +3,8 @@ interface IUser {
   email: string;
   fullName: string;
   displayName: string;
-  imageUrl: string | null | undefined;
-  title: string | null | undefined;
+  imageUrl?: string | null | undefined;
+  title?: string | null | undefined;
 }
 
-interface IUserState {
-  data: IUser | null;
-  isLoading: boolean;
-  isAuthorized: boolean;
-}
-
-export type { IUser, IUserState };
+export type { IUser };
```

Comment:
remove this line. should be `export interface IUser {`
file should be called as IUser